PiwikTracker/Queue.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  EventQueue.swift
+//  PiwikTracker
+//
+//  Created by Cornelius Horstmann on 09.10.16.
+//  Copyright © 2016 Mattias Levin. All rights reserved.
+//
+
+import Foundation
+
+/// An Element that is queueable.
+protocol Queueable {
```

Comment:
I guess you want this so the queue implementation can pull out the data and store it?